securepy/stdcapture.py

```diff
@@ -155,13 +162,16 @@ def capture(self, func: t.Callable, args=None, kwargs=None) -> t.Any:
 
     def override_std(self) -> None:
         """
-        Override `sys.stdout` and `sys.stderr` to use
+        Override `sys.stdout`, `stdin` and `sys.stderr` to use
         `StringIO` instead to capture standard output & error.
         """
         if not isinstance(sys.stdout, StringIO):
             sys.stdout = self.capturing_stdout
         if not isinstance(sys.stderr, StringIO):
             sys.stderr = self.capturing_stderr
+        if not isinstance(sys.stdin, StringIO):
+            if self.stdin:
+                sys.stdin = StringIO(self.stdin)
```

Comment:
We can use a single if statement to make this look a little bit better, lower indent depth is always preferred in python
```suggestion
        if not isinstance(sys.stdin, StringIO) and self.stdin:
            sys.stdin = StringIO(self.stdin)
```